core/cas-server-core-tickets/src/test/java/org/apereo/cas/ticket/registry/BaseTicketRegistryTests.java

```diff
@@ -300,6 +301,22 @@ public void verifyGetExistingTicket() throws Exception {
         assertEquals(ticketGrantingTicketId, ticket.getId(), () -> "Ticket IDs don't match. useEncryption[" + useEncryption + ']');
     }
 
+    /**
+     * Excercise block of code in getTicket that runs when get {@code ticketRegistry.getTicket()} called.
```

Comment:
This would have no effect. You want to use the link tag:

```
{@link TicketRegistry#getTicket()}
```

...for both references.